Makefile

```diff
@@ -1,9 +1,9 @@
 # ################ Assets ################
 
-assets: strings format
+assets: swiftgen format
 
-strings:
-	@(cd BuildTools; SDKROOT=macosx; swift run -c release swiftgen config run --config ./swiftgen.yml)
```

Comment:
I removed a space when copy-pasting the command, I will add it again before merging.